capa/features/extractors/ida/insn.py

```diff
@@ -70,11 +76,15 @@ def extract_insn_api_features(fh: FunctionHandle, bbh: BBHandle, ih: InsnHandle)
     if not insn.get_canon_mnem() in ("call", "jmp"):
         return
 
-    for api in check_for_api_call(fh.ctx, insn):
-        dll, _, symbol = api.rpartition(".")
-        for name in capa.features.extractors.helpers.generate_symbols(dll, symbol):
+    # check calls to imported functions
+    for api in check_for_api_call(insn, get_imports(fh.ctx)):
```

Comment:
what's the type here of `api`? worth splitting up or documenting?

Reply:
I can put a comment here and below to make it more obvious. `check_for_api_call` returns a `tuple`.